docs/tutorials/open-source/apache-flink/0_setup.qmd

```diff
@@ -0,0 +1,35 @@
+# Getting started
```

Comment:
```suggestion
---
execute:
  freeze: auto
---

# Getting started
```

to freeze the output as it gets generated on your machine you can add this to the top of each of the `qmd` files